tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -859,24 +859,38 @@ def _wait_for_traffic(self, producer, acked, timeout=1800):
                 f"Low throughput while preparing for the test: {_throughput}: {e}"
             )
 
-    @ignore
     @cluster(num_nodes=5, log_allow_list=RESTART_LOG_ALLOW_LIST)
     def test_decommission_and_add(self):
-        """
+        """Decommission and add while under load.
+
         Preloads cluster with large number of messages/segments that are also
-        replicated to the cloud, and then runs the decommission-and-add a node
-        stage. This could have been a part of 'test_combo_preloaded' but
-        this stage alone is too heavy and long-lasting, so it's put into a
-        separate test.
+        replicated to the cloud. Then runs the decommission-and-add a node
+        stage.
         """
+
+        self.logger.info(f'verify cluster > 3 nodes')
+        if self._num_brokers <= 3:
+            self.logger.warn('need more than 3 nodes to run test')
+            return
+
+        self.logger.info('verify operator-v2 is not activated')
+        # kubectl get redpanda -n=redpanda
+        get_redpanda = self.redpanda.kubectl.cmd(
+            ['get', 'redpanda', '-n=redpanda'])
+        if len(get_redpanda) > 0:
+            self.logger.warn('cannot run test with operator-v2')
+            return
+
         # create default topics
         self._create_default_topics()
 
         self.adjust_topic_segment_properties(
             segment_bytes=self.min_segment_size,
             retention_local_bytes=2 * self.min_segment_size)
+
         # Generate a realistic number of segments per partition.
         self.load_many_segments()
+        producer = None
         try:
             producer = KgoVerifierProducer(
```

Comment:
from discussion with @travisdowns , future refactor:
* use a `with load` python to bring nominal node or 100% traffic to node.
* move the decomm procedure into the base class